### PR TITLE
fix(EMS-512): Insurance - Start - Fix and add extra wording

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:
* Rectify text on insurance start page to match designs
* Add extra required text for sign in, about the service and quote under start button

### CODE REVIEW

File: e2e-tests/content-strings/pages/insurance/index.js [hunk 1]
```diff
@@ -104,8 +104,32 @@ const START = {
...
+  FIND_OUT_MORE: {
+    BODY_1: 'You can',
+    LINK: {
+      TEXT: 'find out more about UKEF export insurance',
+      HREF: LINKS.EXTERNAL.GUIDANCE,
+    },
+    BODY_2: ", if you want to know who's eligible and what's covered.",
+  },
+  QUOTE: {
+    BODY_1: 'You can also',
```
Comment (ttbarnes) [inline e2e-tests/content-strings/pages/insurance/index.js:126]: Could we be more descriptive instead of BODY_1/2 etc? e.g `YOU_CAN`, `TO_CONTINUE`

Also don't think we should a content string with a comma

File: src/ui/templates/insurance/start.njk [hunk 2]
```diff
@@ -51,4 +51,20 @@
 
   </form>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters-from-desktop">
+        <p class="govuk-body" data-cy="sign-in">
+          <span>{{ CONTENT_STRINGS.SIGN_IN.BODY_1 }}</span> <a class="govuk-link govuk-link--no-visited-state" href="{{ CONTENT_STRINGS.SIGN_IN.LINK.HREF }}" data-cy="sign-in-link">{{ CONTENT_STRINGS.SIGN_IN.LINK.TEXT }}</a> <span>{{ CONTENT_STRINGS.SIGN_IN.BODY_2 }}</span>
+        </p>
```
Comment (ttbarnes) [inline src/ui/templates/insurance/start.njk:58]: Can we have each tag on its own line?

Comment (Zainzzkk) [reply src/ui/templates/insurance/start.njk:58]: it fails the e2e test because it expects a \n then

Comment (ttbarnes) [reply src/ui/templates/insurance/start.njk:58]: Up to you, in other places we data `data-cy` on each element inside a paragraph and test each element individually.

It does complicates the test; The trade off to not doing this is the nunjucks template is hard to follow.

File: e2e-tests/cypress/e2e/journeys/insurance/start.spec.js [hunk 3]
```diff
@@ -69,6 +69,36 @@ context('Insurance Eligibility - start page', () => {
...
+
+    it('renders a "get a quote" body text', () => {
+      const { QUOTE } = CONTENT_STRINGS;
+
+      cy.checkText(insurance.startPage.getAQuote.youCan(), QUOTE.YOU_CAN);
+
+      cy.checkLink(insurance.startPage.getAQuote.link(), QUOTE.LINK.HREF, QUOTE.LINK.TEXT);
+
+      cy.checkText(insurance.startPage.getAQuote.ifYouNeed(), QUOTE.IF_YOU_NEED);
+    });
```
Comment (ttbarnes) [inline e2e-tests/cypress/e2e/journeys/insurance/start.spec.js:101]: Not a blocker, could destructure the sections from `insurance.startPage`

**Review by ttbarnes — APPROVED** [review]

File: e2e-tests/cypress/e2e/journeys/insurance/start.spec.js (hunk 3 above)
Comment (Zainzzkk) [reply e2e-tests/cypress/e2e/journeys/insurance/start.spec.js:101]: will do in next PR
